docs/guide/docs/manage/configuration.md

```diff
@@ -0,0 +1,123 @@
+- Error: there was an error that should be addressed
+
+By default, you will see `debug` with all other log levels in the console, and `errors` will be saved in the database (useful to keep track of them).
+When you start Botpress in `production` mode, `debug` logs will be disabled for better performances.
+
+It is also possible to send log output to a file in a specific folder. Check below for the required configuration
+
+### How to save logs on the file system
+
+Edit your `botpress.config.json` file and change your file to match the following:
+
+```js
+{
+  ...
+  "logs": {
+    ...
+    "fileOutput": {
+      "enabled": true,
+      "folder": "./", // Change this to any folder, by default it will be in the same folder as the executable
+      "maxFileSize": 10000 // By default, the maximum file size will be kept under 10mb
+    }
+  },
+}
+```
+
+## Enable or disable modules
+
+When you start Botpress for the first time, the most popular modules included with the binary will be added to your `botpress.config.json` file. If you want to disable or enable modules, you need to edit the `modules` option.
+
+The string `MODULE_ROOT` is a special string that is replaced when your configuration file is read. It represents the location of the modules folder on your hard drive, you shouldn't have to change it most of the times.
+
+```js
+{
+  ..."modules": [
+    {// When you new modules, you need to set their location, and if they are enabled or not.
+      "location": "MODULES_ROOT/analytics",
+      "enabled": true // You can change this to false to disable the module.
+    },
+    {
+      "location": "MODULES_ROOT/basic-skills",
+      "enabled": true
+    }
+}
+```
+
+# Individual Bot Configuration
+
+Every bot that you create will have its own configuration file. It is located at `data/bots/NAME_OF_BOT/bot.config.json`. Most of the available options can be edited by clicking on the `Config` link next to the bot name on the administration panel.
+
+If you enable additional modules that adds other
+
+# Module Configuration
+
+When you enable a module on Botpress, they are available globally, which means that you can't disable or enable them for specific bots. You can, however, use a different configuration for every bots.
+
+Each module has a different set of possible configuration, so we won't go through them here. What you need to know is that when you run a module for the first time, the default configuration will be created in `data/global/config/MODULE_NAME.json`. If you need a special configuration for your bot, create a `config` folder in the bot folder, and copy the file in your bot folder: `data/bots/BOT_NAME/config/MODULE_NAME.json`. You will probably need to create the folder the first time.
+
+# Environment Variables
+
+There are some variables that **cannot be set** in the `botpress.config.json` file. These must be either defined as environment variables, or defined in a `.env` file in the same folder as the Botpress executable.
```

Comment:
all botpress.config.json config can also be set by env var. I think this could be relevant to add here